packages/flutter/lib/src/material/chip.dart

```diff
@@ -1011,6 +1009,9 @@ class _RawChipState extends State<RawChip>
   late CurvedAnimation enableAnimation;
   late CurvedAnimation selectionFade;
 
+  late final WidgetStatesController statesController =
+      WidgetStatesController()..addListener(() => setState(() {}));
```

Comment:
Should this be moved after the `.update` calls in `initState`?

Reply:
Sure, I can do that !

Reply:
@MitchellGoodwin I made the changes in [Add listener after updates](https://github.com/flutter/flutter/pull/161487/commits/4eed83057ffbe85d4763960b3a06939e028b724e)